edx_proctoring/views.py

```diff
@@ -1032,24 +1032,28 @@ class BackendUserManagementAPI(AuthenticatedAPIView):
     """
     Manage user information stored on the backends
     """
-    @method_decorator(require_staff)
     def delete(self, request, user_id):  # pylint: disable=unused-argument
```

Comment:
Any chance we can change this to a POST since it's not actually deleting a resource from this service?